packages/netstring/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "@endo/netstring",
+  "version": "0.1.0",
+  "description": "Description forthcoming.",
+  "keywords": [],
+  "author": "Endo contributors",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/endojs/endo/tree/master/packages/netstring#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/endojs/endo.git"
+  },
+  "bugs": {
+    "url": "https://github.com/endojs/endo/issues"
+  },
+  "type": "module",
+  "parsers": {
+    "js": "mjs"
+  },
+  "main": "./dist/netstring.cjs",
+  "module": "./index.js",
+  "browser": "./dist/netstring.umd.js",
+  "unpkg": "./dist/netstring.umd.js",
+  "types": "./index.d.ts",
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "import": "./index.js",
+      "require": "./dist/netstring.cjs",
+      "browser": "./dist/netstring.umd.js"
+    },
+    "./reader": {
+      "import": "./reader.js",
+      "require": "./dist/netstring-reader.cjs",
+      "browser": "./dist/netstring-reader.umd.js"
+    },
+    "./writer": {
+      "import": "./writer.js",
+      "require": "./dist/netstring-writer.cjs",
+      "browser": "./dist/netstring-writer.umd.js"
+    }
+  },
+  "scripts": {
+    "build": "rollup --config rollup.config.js",
+    "clean": "rm -rf dist",
+    "cover": "nyc ava",
```

Comment:
what's the current coverage? Do we track it somewhere? Do we run this in ci? I guess I could check.

Reply:
This is one of those things that’s supposed to work but doesn’t seem to, and is also carried forward by our scaffold. I get a broad table of zeroes from running `cover`. Since this is present in every package, I would like to address separately. https://github.com/endojs/endo/issues/694